src/test/scala/org/zalando/spark/jsonschema/SparkTestEnv.scala

```diff
@@ -15,4 +17,13 @@ object SparkTestEnv {
       .getOrCreate()
   }
 
+  def getTestResourceContent(relativePath: String): String = {
+    val inputStream = getClass.getResourceAsStream(relativePath)
+    try Source.fromInputStream(inputStream).mkString
+    catch {
+      case _: NullPointerException =>
+        throw new NullPointerException(s"No content found in $relativePath")
```

Comment:
NullPointerException is thrown by the system. I'd suggest to use RuntimeException or IllegalArgumentException, or something similar.